src/asynk/async_worker_pool.rs

```diff
@@ -8,19 +8,27 @@ use log::error;
 use tokio::task::JoinHandle;
 use typed_builder::TypedBuilder;
 
+/// A executioner of tasks, it executes tasks only of one given task_type, it sleeps when they are
```

Comment:
I think this doc is related to worker, not worker pool

Reply:
[move comment to worker](https://github.com/ayrat555/fang/pull/104/commits/44e833b300f6f60ddc946ec23c0e87bc34ecd6ec)